libraries/AP_Notify/Display.cpp

```diff
@@ -326,7 +323,7 @@ bool Display::init(void)
 
     // initialise driver
     for(uint8_t i=0; i<8 && _driver == nullptr; i++) {
-        if (! (I2C_BUS_PROBE_MASK & (1<<i))) {
+        if (! (hal.i2c_mgr->get_i2c_bus_mask_external() & (1<<i))) {
```

Comment:
This doesn't work on the Cube because both buses are external.

Reply:
it does work in exactly the way it worked before (with the both_external boolean)
I didn't change the mask for the cube as it would change the bus order, so would break peoples configs

Reply:
Where is the both_external boolean? I think you are confusing Compass with Display...

Reply:
ahh, yes, sorry, I thought you meant compass. You're right that this will break display on i2c0 on cube

Reply:
fixed